ghost/admin/app/components/posts/debug.hbs

```diff
@@ -67,29 +67,37 @@
+                        {{#if failure.member.id}}
+                            <LinkTo @route="member" @model={{failure.member.id}} class="gh-email-debug-member">
+                                <GhMemberAvatar @member={{failure.member.record}} @containerClass="w9 h9 mr3 flex-shrink-0" />
+                                <div>
+                                    <h3 class="ma0 pa0 gh-members-list-name">{{failure.recipient.name}}</h3>
+                                    <p class="ma0 pa0 middarkgrey f8 gh-members-list-email">{{failure.recipient.email}}</p>
+                                </div>
+                            </LinkTo>
+                        {{else}}
+                            <div class="gh-email-debug-member">
+                                <figure class="gh-member-gravatar w9 h9 mr3 flex-shrink-0 bg-red">
+                                    <div class="gh-member-initials flex w9 h9 items-center justify-center br-100">
+                                        <span class="gh-member-avatar-label gh-member-list-avatar">{{failure.recipient.initials}}</span>
+                                    </div>
+                                </figure>
+                                <div>
+                                    <h3 class="ma0 pa0 gh-members-list-name">{{failure.recipient.name}}</h3>
+                                    <p class="ma0 pa0 middarkgrey f8 gh-members-list-email">{{failure.recipient.email}}</p>
                                 </div>
-                            </figure>
-                            <div>
-                                <h3 class="ma0 pa0 gh-members-list-name">{{failure.recipient.name}}</h3>
-                                <p class="ma0 pa0 middarkgrey f8 gh-members-list-email">{{failure.recipient.email}}</p>
                             </div>
-                        </div>
+                        {{/if}}
                     </div>
                     <div class="gh-list-data gh-email-debug-col-failure">
                         <div class="gh-email-debug-failure-details">
                             <div class="gh-email-debug-failure-codes">
                                 <span class="gh-email-debug-failure-code">Failure code: <span>{{failure.code}}</span></span>
-                                <span class="gh-email-debug-failure-code">Enhanced code: <span>{{failure.enhancedCode}}</span></span>
-                            </div>
-                            <div class="gh-email-debug-readmore-error">
-                                <input type="checkbox" id="error-{{index}}-checkbox" class="toggle-checkbox">
-                                <label for="error-{{index}}-checkbox">{{svg-jar "dotdotdot"}}</label>
-                                <span>{{failure.message}}</span>
+                                {{#if failure.enhancedCode}}
+                                    <span class="gh-email-debug-failure-code">Enhanced code: <span>{{failure.enhancedCode}}</span></span>
```

Comment:
@peterzimon Don't think we need another `span` here inside the original one right ?

Reply:
we kinda do to format the enhanced code value (this is valid HTML fwiw)

Reply:
Aah cool, TIL 😄